scripts/hist_importer.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python3
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2017 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
+# Copyright 2014-2017 Josefson Souza <josefson.br@gmail.com>
+
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+
+"""Tool to import browser history from other browsers."""
+
+
+import argparse
+import sqlite3
+import sys
+import os
+
+
+def parse():
+    """Parse command line arguments."""
+    description = ("This program is meant to extract browser history from your"
+                   "previous browser and import them into qutebrowser.")
+    epilog = ("Databases:\n\tQutebrowser: Is named 'history.sqlite' and can be"
```

Comment:
This block is a bit of a mess, here's a possible fix:

`'''` or `"""` will begin/end a python string that gets newlines automatically, so

```
'''
One 
Two
Three
'''
``` 
would keep those newlines and whitespace. I'm not sure if the lines will fit in the line limit though.

Reply:
Are you talking about the ```description``` variable or also about ```epilog```?
If it's about the epilog, don't you think it rather be messy for the code instead of being messy for the user depending on his terminal frame size?

Reply:
Using a multi-line string only works correctly here if you wouldn't use `argparse.RawTextHelpFormatter`, but indeed that seems to prevent having newlines between blocks here. So I guess this is fine.

Reply:
Also, please lower-case `qutebrowser` here.